settings.gradle.kts

```
@@ -162,7 +162,8 @@ include(":instrumentation:internal:internal-reflection:javaagent-integration-tes
 include(":instrumentation:internal:internal-url-class-loader:javaagent")
 include(":instrumentation:internal:internal-url-class-loader:javaagent-integration-tests")
 include(":instrumentation:couchbase:couchbase-2.0:javaagent")
-include(":instrumentation:couchbase:couchbase-2.0:javaagent-unit-tests")
+include(":instrumentation:couchbase:couchbase-2.0-common:javaagent")
+include(":instrumentation:couchbase:couchbase-2.0-common:javaagent-unit-tests")
```

similar to `log4j-context-data-2-common`
```suggestion
include(":instrumentation:couchbase:couchbase-2-common:javaagent")
include(":instrumentation:couchbase:couchbase-2-common:javaagent-unit-tests")
```